crates/nu-command/tests/commands/umkdir.rs

```diff
@@ -2,6 +2,22 @@ use nu_test_support::fs::files_exist_at;
 use nu_test_support::playground::Playground;
 use nu_test_support::{nu, pipeline};
 
+#[cfg(test)]
+#[cfg(not(windows))]
```

Comment:
The `#[cfg(test)]` is probably not needed as this should be a test only file, so the problem was down to the requirements on the windows cases.

```suggestion
#[cfg(not(windows))]
```